content/cn/use_cases.html

```diff
@@ -383,11 +383,11 @@ <h2 id="增量处理管道">增量处理管道</h2>
 
 <h2 id="dfs的数据分发">DFS的数据分发</h2>
 
-<p>Hadoop的一个流行用例是压缩数据，然后将其分发回在线服务存储层，以供应用程序使用。
+<p>一个常用场景是先在Hadoop上处理数据，然后将其分发回在线服务存储层，以供应用程序使用。
 例如，一个Spark管道可以<a href="https://eng.uber.com/telematics/">确定Hadoop上的紧急制动事件</a>并将它们加载到服务存储层(如ElasticSearch)中，供Uber应用程序使用以增加安全驾驶。这种用例中，通常架构会在Hadoop和服务存储之间引入<code class="highlighter-rouge">队列</code>，以防止目标服务存储被压垮。
-对于队列的选择，一种流行的选择是Kafka，这个模型经常导致__在DFS上存储相同数据的冗余(用于计算结果的离线分析)和Kafka(用于分发)__</p>
+对于队列的选择，一种流行的选择是Kafka，这个模型经常导致<strong>在DFS上存储相同数据的冗余(用于计算结果的离线分析)和Kafka(用于分发)<strong></p>
```

Comment:
ditto

Reply:
In markdown, we use __ as bold, but we should use <strong> </ strong> tags in html.